#!/usr/bin/env python
PACKAGE = "victor_driver"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("wheel_diameter", double_t, 0, "Drive Wheel Diameter (m)", 0.2, 0.0, 1.0)
gen.add("wheel_track", double_t, 0, "Distance Between Drive Wheels (m)", 0.5, 0.0, 2.0)


exit(gen.generate(PACKAGE, "victor_base_configure", "BaseController"))
